open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -323,22 +330,219 @@ DEFINE
+ SLTYPELENGTH              = NUMBER
+ SLTYPESMAX                = NUMBER
+ TESTACCOUNT               = NUMBER
+ TESTACCTLIST              = CHARACTER ARRAY(10)
+ TESTACCTLISTCOUNT         = NUMBER
+ TESTACCTLISTMAX           = 10
+ TESTERRORCODE             = NUMBER
+ TESTMODE                  = NUMBER
+ TESTNAMECITY              = CHARACTER
+ TESTNAMEEXTRAADDRESS      = CHARACTER
+ TESTNAMEID                = CHARACTER
+ TESTNAMEIDTYPE            = NUMBER
+ TESTNAMELOCATOR           = NUMBER
+ TESTNAMESHORTNAME         = CHARACTER
+ TESTNAMESSN               = CHARACTER
+ TESTNAMESTATE             = CHARACTER
+ TESTNAMESTREET            = CHARACTER
+ TESTNAMEZIPCODE           = CHARACTER
+ TMPLOC                    = NUMBER
+ TMPNUM                    = NUMBER
+ TRUEFALSECHR              = CHARACTER ARRAY(1)
+ VALIDLOANNAMETYPES        = NUMBER ARRAY(99)
+ VALIDSHARENAMETYPES       = NUMBER ARRAY(99)
+ VALIDNAMETYPESCHR         = CHARACTER
+ VALIDNAMETYPES            = NUMBER ARRAY(99)
+
+ BNODELIMITER              = NUMBER
+ BNODELIMNONE              = 0
+ BNODELIMQUOTE             = 34
+ BNODELIMCR                = 13
+ BNODELIMLF                = 10
+ BNODELIMCOMMA             = 44
+ BNODELIMTAB               = 09
+ BNODELIMVB                = 124
+ BNODELIMCARET             = 94
+ BNODELIMPIPE              = 124
+ BNODELLIMSEMICOLON        = 59
+
+ ERRORCODEDESCR            = CHARACTER ARRAY(1,19)
+ ECDEFAULT                 = 0
+ ECCUSTOM                  = 1
+
+ NAMETYPESMAX              = 53
+
+ COPYNAMELIST              = CHARACTER ARRAY(36,20)
+ COPYNAMELISTERROR         = CHARACTER ARRAY(20)
+ CNLERRORCHR               = CHARACTER
+ COPYNAMELISTCOUNT         = NUMBER
+ COPYNAMECOUNTMAX          = NUMBER
+ CNLSSN                    = 0
+ CNLLAST                   = 1
+ CNLFIRST                  = 2
+ CNLSTREET                 = 3
+ CNLEXTRAADDRESS           = 4
+ CNLCITY                   = 5
+ CNLSTATE                  = 6
+ CNLZIPCODE                = 7
+ CNLLOC                    = 8
+ CNLMBRADDRNUMLINK         = 9
+ CNLMEMBERNUMLINK          = 10
+ CNLTYPE                   = 11
+ CNLSOURCE                 = 12
+ CNLBIRTHDATE              = 13
+ CNLLOCATOR                = 14
+ CNLSHORTNAME              = 15
+ CNLLONGNAME               = 16
+ CNLADDRESSTYPE            = 17
+ CNLSUBTYPE                = 18
+ CNLNAMEFORMAT             = 19
+ CNLSSNTYPE                = 20
+ CNLSSNOVERRIDE            = 21
+ CNLSSNCERTIFICATION       = 22
+ CNLFOREIGNTIN             = 23
+ CNLUSPERSONFLAG           = 24
+ CNLTITLE                  = 25
+ CNLMIDDLE                 = 26
+ CNLSUFFIX                 = 27
+ CNLPHONETYPE              = 28
+ CNLHOMEPHONE              = 29
+ CNLCARRIERROUTE           = 30
+ CNLCOUNTRYCODE            = 31
+ CNLCOUNTRY                = 32
+ CNLRESTRICT               = 33
+ CNLLASTADDRVERIFDATE      = 34
+ CNLSEX                    = 35
+ CNLDUPLICATED             = 36
+
+ CNTYPE                    = NUMBER
```

Comment:
I don't see CNTYPE used in the code. If you don't plan to used it in the future, I suggest removing it.